build_runner/lib/src/logging/build_for_input_logger.dart

```diff
@@ -60,6 +60,15 @@ class ErrorRecordingLogger implements Logger {
     if (logLevel >= Level.SEVERE) {
       _errorWasSeen = true;
     }
+    if (logLevel >= Level.WARNING) {
```

Comment:
I just realized this method should never have worked... including the `_errorWasSeen` tracking. Are we missing tests?

Since we have a `_delegate` rather than `extends` the calls to `_delegate.sever()` never hit this method..

Reply:
Ah, because we also set `_errorWasSeen` directly in the call to `severe` :)